api/python/quilt3/api.py

```diff
@@ -1,11 +1,11 @@
-from .data_transfer import copy_file, get_bytes, delete_url, list_url
-from .packages import Package
+from .backends import get_package_registry
+from .data_transfer import copy_file
 from .search_util import search_api
 from .util import (QuiltConfig, QuiltException, CONFIG_PATH,
```

Comment:
minor: at this scale, one import per line reads better

Reply:
Let's fix with [isort](https://isort.readthedocs.io/en/stable/) later.